Tools/setup_gazebo.bash

```diff
@@ -7,22 +7,19 @@
 #
 # License: according to LICENSE.md in the root directory of the PX4 Firmware repository
 
-if [ "$#" != 2 ]
-then
-    echo usage: source setup_gazebo.bash src_dir build_dir
-    echo ""
+if [ "$#" != 2 ]; then
+    echo -e "usage: source setup_gazebo.bash src_dir build_dir\n"
     return 1
 fi
 
 SRC_DIR=$1
 BUILD_DIR=$2
 
 # setup Gazebo env and update package path
-export GAZEBO_PLUGIN_PATH=${BUILD_DIR}/build_gazebo:${GAZEBO_PLUGIN_PATH}
-export GAZEBO_MODEL_PATH=${GAZEBO_MODEL_PATH}:${SRC_DIR}/Tools/sitl_gazebo/models
-# Disabling the remote model download seems only necessary with Gazebo 6
-#export GAZEBO_MODEL_DATABASE_URI=""
-export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:${SRC_DIR}/Tools/sitl_gazebo/Build/msgs/:${BUILD_DIR}/build_gazebo
```

Comment:
`Build/msgs` didn't exist be this PR and msgs aren't needed to be in the path variable